tools/validate_motion.py

```
@@ -160,7 +160,7 @@ def validate_feed(video_count):
 
 def validate_bundled():
     if not os.path.isfile(BUNDLED_MANIFEST):
-        warn(True, "bundled manifest not found (expected at shift/app/src/main/assets/)")
+        warn(False, "bundled manifest not found (expected at shift/app/src/main/assets/)")
```

Ruff (0.16.2)

[warning] Boolean positional value in function call

(FBT003)